Formula/ksync.rb

```diff
@@ -0,0 +1,27 @@
+class Ksync < Formula
+  desc "Sync files between your local system and a kubernetes cluster"
+  homepage "https://ksync.github.io/ksync/"
+  url "https://github.com/ksync/ksync.git",
+    :tag  => "0.4.5"
+  sha256 "bd47c5b5418bfc6d2f08abe855a52d4189f8bf380747cdd4e5c1a3ca0a87498a"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # Set some stamps and build local CLI binary
+    time = `date +%Y%m%dT%H%M%S`.chomp
+    goversion = `go version | awk '{ print $3 }'`
+    commit = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
+    system "go", "build", "-ldflags",
+              "-w -X github.com/ksync/ksync/pkg/ksync.GitCommit=#{commit} -X github.com/ksync/ksync/pkg/ksync.GitTag=#{version} -X github.com/ksync/ksync/pkg/ksync.BuildDate=#{time} -X github.com/ksync/ksync/pkg/ksync.VersionString=Homebrew -X github.com/ksync/ksync/pkg/ksync.GoVersion=#{goversion}",
```

Comment:
Audit error about the length of this line, but it needs to be passed as a _single_ argument to `-ldflag` to work. Suggestions?

Reply:
Everywhere you have a space you can also do `",\n"` that should fix it.

Reply:
You could also try something like:
```
"-w -X github.com/ksync/ksync/pkg/ksync.GitCommit=#{commit}" \
"-X github.com/ksync/ksync/pkg/ksync.GitTag=#{version}" \
"-X github.com/ksync/ksync/pkg/ksync.BuildDate=#{time}" \
"-X github.com/ksync/ksync/pkg/ksync.VersionString=Homebrew" \
"-X github.com/ksync/ksync/pkg/ksync.GoVersion=#{goversion}"
```

The `\` operator will concatenate the multiple lines into a single argument. You could also try something along the lines of:
```
ldflags = %W[
  -w -X github.com/ksync/ksync/pkg/ksync.GitCommit=#{commit}
  -X github.com/ksync/ksync/pkg/ksync.GitTag=#{version}
  -X github.com/ksync/ksync/pkg/ksync.BuildDate=#{time}
  -X github.com/ksync/ksync/pkg/ksync.VersionString=Homebrew
  -X github.com/ksync/ksync/pkg/ksync.GoVersion=#{goversion}
]

[...]

system "go", "build", "-ldflags", ldflags.join(" ")
```